Feature/organizer dashboard

dducwsxuaan opened the pull request:
I created a page for displaying the dashboard for organizers. However, it just used the mock data from data/_mock_db

AndrwPham committed:
Added: Sign-up backend wiring and fixed errors after schema changes

AndrwPham committed:
Added: Login backend wiring

AndrwPham committed:
refactor: Auth endpoint

AndrwPham committed:
refactor: jwt.strat for cookies

AndrwPham committed:
Added: AuthContext for state management

AndrwPham committed:
Added: Navbar changed dynamically when login and logout

AndrwPham committed:
Added: Syncfusion Liciense Key

phuclhp1922 committed:
feat: display user's email on profile

phuclhp1922 committed:
feat: display user's email on profile

phuclhp1922 committed:
feat: display user's email on profile

phuclhp1922 committed:
feat: display username or user's email conditionally

phuclhp1922 committed:
fix: Add username when sign-up

fuisl committed:
feat: accept either username or email for login.

fuisl committed:
test: update tests for login with username or email.

fuisl committed:
Merge pull request #35 from AndrwPham/fix/auth

Fix/auth

fuisl committed:
feat: dtos for user info service

fuisl committed:
feat: allow changing password.

fuisl committed:
feat: change email

fuisl committed:
feat: implement controller for update info.

fuisl committed:
Merge branch 'feat/query-user-info' into dev

dducwsxuaan committed:
Create dashboard for organizer with mock data

AndrwPham commented:
@dducwsxuaan Before proceeding to merge, can you changes to files following the copilot suggestion

dducwsxuaan committed:
Modify the dashboard page as Copilot suggested and create more mock data

dducwsxuaan commented:
@AndrwPham I changed the code as you requested and added more mock data. Please check them. Also, the Load more button appears when the number of events shown on the Home page is 9, but when clicking the button, more events are shown on the Home page, not on another page.
